kafka-connect/src/main/java/io/tabular/iceberg/connect/api/Committer.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.tabular.iceberg.connect.api;
+
+public interface Committer {
+  void commit(CommittableSupplier committableSupplier);
+}
```

Comment:
This is the Committer interface I'm proposing. 
I've taken some inspiration from Flink here (they have a concept of Committable as well).  

I am not making this interface pluggable in this PR, nor do I intend to do so any time soon until the need arises. 
This means we can still change it after this PR without having to do a breaking release. 
So we don't need to get this interface 100% right here but we should at least try to align directionally on the design of the interface.

Reply:
What is the purpose of having a pluggable committer interface?

Reply:
Answered offline.